README.rst

```diff
@@ -62,6 +62,20 @@ like the following::
     }
 
 
+-----
+Goals
+-----
+
+As a Django REST Framework JSON API (short DJA) we are trying to address following goals:
+
+1. Support the [JSON API](http://jsonapi.org/) spec to compliance
+2. Be as compatible with Django REST Framework as possible
+   e.g. issues in Django REST Framework should be fixed upstream and not worked around in DJA
```

Comment:
This line looks good in the plain text, but I think it won't format well in the final documentation. It would look like "... as possible e.g. issues in Django...." I think the line should be wrapped with parentheses.